python/cugraph/cugraph/tests/mg/test_mg_renumber.py

```diff
@@ -287,3 +289,57 @@ def test_mg_renumber_common_col_names(graph_file, dask_client):
     assert renumber_map.renumbered_dst_col_name != "dst"
     assert renumber_map.renumbered_src_col_name in renumbered_df.columns
     assert renumber_map.renumbered_dst_col_name in renumbered_df.columns
+
+
+@pytest.mark.skipif(
+    is_single_gpu(), reason="skipping MG testing on Single GPU system"
+)
+def test_pagerank_string_vertex_ids(dask_client):
+    """
+    Ensures string vertex IDs can be used.
+
+    Note: the dask_client fixture sets up and tears down a LocalCUDACluster.
+    See ../conftest.py
+    """
+    # Use pandas and to_csv() to create a CSV file that can be read in by both
+    # dask_cudf and cudf.
+    df = pandas.DataFrame({"src": ['a1', 'a1', 'a2', 'a3'],
+                           "dst": ['a2', 'a3', 'a4', 'a4'],
+                           }
+                          )
+    tempdir_object = tempfile.TemporaryDirectory()
```

Comment:
Is this mechanism necessary?  Isn't it possible to take a pandas Dataframe and convert it to a dask Dataframe directly without going to disk first?

Reply:
yes, I debated if we should do it this way in the tests or if we should just go directly from an inline pandas/cudf DataFrame to a dask_cudf one (as is done elsewhere).  I ultimately did it this way to match the user's script [here](https://github.com/rapidsai/cugraph/issues/2686), just to be thorough. However, I can remove this in favor of our standard mechanism which doesn't involve a temp file.

Reply:
On second thought, I think I will simplify the test and not use a tempfile with `read_csv()`